addons/insurgents/config.cpp

```diff
@@ -0,0 +1,26 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {
+            QGVAR(soldier_i),
+            QGVAR(soldier_o),
+            QGVAR(soldier_b)
+        };
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {
+            "tac_common"
```

Comment:
```suggestion
            "tac_main"
```
There is no `tac_common`. :)

Reply:
woops, just went with find and replace, testing this in game right now if works.